minimal-py-app/app.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask, make_response
+
+app = Flask(__name__)
+
+@app.route("/errorz")
+def errorz():
+	return "Something broke!", 500
```

Comment:
Please add a docstring explaining what this function is accomplishing, along with the return type

Reply:
Okay.